Update the documentation link in CONTRIBUTING.rst

birdcar opened the pull request:
### Summary

As I was setting up a local dev environment for Kolibri, I found that the current link to Kolibri's documentation (introduced in 62974dc) leads potential contributors to a 404:
https://kolibri-dev.readthedocs.io/en/develop/start/contributing/index.html

This Pull Request will update the link so that it points to the main "Contributing" page here: https://kolibri-dev.readthedocs.io/en/develop/contributing/index.html

### Reviewer guidance

This shouldn't be a risky change (unless I've missed something) and the change can be tested by [clicking the link in my branches rendered `CONTRIBUTING.rst` file](https://github.com/nickcannariato/kolibri/blob/docs/fix-contributing-url/CONTRIBUTING.rst): 

---

### Contributor Checklist

PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

birdcar committed:
docs: fix URL in CONTRIBUTING.rst

Remove the unnecessary '/start/' from the link to the CONTRIBUTING
documentation.

This will ensure that users that would like to contribute to Kolibri are
directed to the right place and don't encounter a 404.

indirectlylit reviewed (approved):
thanks! I must have missed this when I flattened the docs hierarchy. Made a new issue to prevent this sort of thing in the future, because it's happened before: #5146

In the readme, I actually took the strategy of just linking to the top-level page, precisely to avoid broken links like this: https://github.com/learningequality/kolibri/pull/4536/files#diff-88b99bb28683bd5b7e3a204826ead112

It's a little less user-friendly short term, but probably safer until we add dead link checking. Either way is fine with me, but feel free to update.

birdcar committed:
docs:Point potential contributors to main docs URL

This is a temporary solution until we can add dead link checks to the
documentation.

related: #5146
Signed-off-by: Nick Cannariato <434063+nickcannariato@users.noreply.github.com>

birdcar commented:
> In the readme, I actually took the strategy of just linking to the top-level page, precisely to avoid broken links like this

@indirectlylit That definitely sounds like a good solution for the time being. I've updated the URL to point to the main page of the developer documentation and also updated the language so that it matches the README exactly now.

EDIT: 
As a side note, it's showing that the changes are approved but I don't have the ability to hit the merge button. Let me know if there's anything else I need to do on this one, otherwise one of you lovely folks will need to hit that merge button for me. 🙇

rtibbles commented:
Yeah, we limit who can push to the repo. Merging, thanks for the contribution!
